src/k8s/pkg/k8sd/app/util.go

```diff
@@ -0,0 +1,93 @@
+package app
```

Comment:
We already have some networking utils in https://github.com/canonical/k8s-snap/tree/main/src/k8s/pkg/utils , wouldn't it be better to put these  there?

Reply:
Done